### ISSUE TITLE
organise the log files

### ISSUE THREAD

**Issue by PatGendre** [fabmob/e-mission-phone-fabmob#19]
- move logs from /var/tmp to /var/log/emission in 2 directories : one for webserver, one for pipeline
- check if this can be done for grfmap logs as well
- name the log file by date (easier for finding data and more gdpr compliant)
- merge the several intake logs into one file (easier to search)
as this could be done using the python logging lib, this can be proposed as a contrib to e-mission.

**Comment by DO3B** [comment]
I changed the RotatingFileHandler to TimedRotatingFileHandler with a backup count of 90 (since we said in the consent that we keep 90 days of data). The handler is configured to rotate at midnight and after some tests, it works (https://github.com/fabmob/e-mission-server-fabmob/commit/adbf60b7d3535127ec19951aa270597539062c77). 

<img width="483" alt="Capture d’écran 2019-07-17 à 09 21 49" src="https://user-images.githubusercontent.com/17317703/61355309-62df8800-a874-11e9-97a4-c0dd4c768305.png">

> **WARNING**: Do not forget to create the folders before, Python will not create them for us.

**Comment by PatGendre** [comment]
good first step :+1:

**Comment by DO3B** [comment]
As for the pipeline, `e-mission` is loading `intake.conf` n (each process) + 1 (intake_launcher) times. We are running the pipeline around 2 AM so we can use the same `when` (midnight) than the webserver's logs but to have a unique file for each pipeline, I think that I will need to modify `e-mission` to achieve that but I can eventually create a script which will be executed around 7 or 8 AM to merge them into a single file and create a daily log.

**Comment by PatGendre** [comment]
Ok, yes it seems a good idea to merge the intake log files with a cron a few hours later after the pipeline is launched.

**Comment by DO3B** [comment]
I merged my work on the logs and applied it to the server (https://github.com/fabmob/e-mission-server-fabmob/pull/2). The logs will now be placed into `/var/log/emisssion`. I added a script (written in python as you asked) into `em-scripts/` (locally not in git for now) which will merge the `intake_<thread>.log` into `intake_global.log`. The script can be optimised but I want to try it with the next pipeline execution so I added an entry into the crontab and the script will be executed at 7 AM tomorrow.
